ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/parts/base/BaseView.java

```diff
@@ -69,169 +52,12 @@ public void onBlur(BlurEvent event) {
         }
     };
 
-    public BaseView(PartStackUIResources resources) {
-        this.resources = resources;
-
+    public BaseView() {
         container = new DockLayoutPanel(Style.Unit.PX);
         container.getElement().setAttribute("role", "part");
         container.setSize("100%", "100%");
         container.getElement().getStyle().setOutlineStyle(Style.OutlineStyle.NONE);
         initWidget(container);
-
-        toolBar = new DockLayoutPanel(Style.Unit.PX);
-        toolBar.addStyleName(resources.partStackCss().ideBasePartToolbar());
-        toolBar.getElement().setAttribute("role", "toolbar");
-        toolBar.addDomHandler(event -> {
-            //activate last focused element if user clicked on part header
-            if (lastFocused != null) {
```

Comment:
Take a note, that this may affect a panel behavior, when you click on part title and last focused element will lose own focus.